pom.xml

```diff
@@ -851,6 +851,7 @@
                 <groupId>org.projectlombok</groupId>
                 <artifactId>lombok</artifactId>
                 <version>1.18.20</version>
+                <scope>provided</scope>
```

Comment:
I don't think is ok to add this additional library. Here is explain the policy of adding new third-party libraries. 
https://alfresco.atlassian.net/wiki/spaces/TECH/pages/248054489/Policy+for+including+third-party+software+in+Alfresco+s+softwarehttps://alfresco.atlassian.net/wiki/spaces/TECH/pages/398850/Third+Party+Licensing
I think all the changes that use Lombok for the code that is delivered to the customers should be reverted.
@tpage-alfresco  what are your thoughts?

Reply:
I thought that "provided" meant that the dependency won't be included in the final artifact?  (but will instead be "provided" by the environment)  However maybe this should be changed to "compile" by default?

Reply:
Actually I remembered this slightly wrong:
> Basically the difference between provided and compile is that provided dependencies are not transitive.
So "provided" is weaker than "compile" - I think this looks fine then.